extensions/sdk/src/__tests__/Rally.test.ts

```diff
@@ -113,6 +113,9 @@ describe('Rally SDK', function () {
     let pausedCallbackCalled = false;
     let resumeCallbackCalled = false;
 
+    browser.tabs.query = jest.fn().mockReturnValueOnce([]);
```

Comment:
You may want to add beforeEach / afterEach and do jest.resetAllMocks or we can have cross failures carried over to other tests I believe.

Reply:
This PR adds `jest.clearAllMocks()` to `beforeEach`, is that sufficient?